docs/cicd/cicd-spans.md

```diff
@@ -0,0 +1,124 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Spans
+--->
+
+# Semantic conventions for CICD spans
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [CICD Spans](#cicd-spans)
+  - [Pipeline run](#pipeline-run)
+  - [Pipeline task run](#pipeline-task-run)
+
+<!-- tocstop -->
+
+## CICD Spans
+
+The conventions described in this section are specific to Continuous Integration / Continuous Deployment (CICD) systems.
+
+Any resources of the [CICD and VCS resource conventions][cicdres] that apply SHOULD be used.
+
+[cicdres]: /docs/resource/cicd.md (CICD and VCS resource conventions)
+
+### Pipeline run
+
+For all pipeline runs, a span with kind `SERVER` SHOULD be created corresponding to the execution of the pipeline run.
+
+The span name MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.43.0/specification/trace/api.md#span).
+
+The span name SHOULD be `{action} {pipeline}` if there is a (low-cardinality) `pipeline` available.
+If there is no (low-cardinality) `{pipeline}` available, the span name SHOULD be `{action}`.
+
+The `{action}` SHOULD be one of the following values: `BUILD`, `RUN`, `SYNC`. If none of these values applies a custom value MAY be used.
+
+The `{pipeline}` SHOULD be the [`cicd.pipeline.name`](/docs/attributes-registry/cicd.md).
```

Comment:
nit: do you need a placeholder for `{pipeline}` given it's always `cicd.pipeline.name` ?

```suggestion
The span name SHOULD be `{action} {cicd.pipeline.name}` if there is a (low-cardinality) pipeline name available.
If pipeline name is not available or is likely to have high cardinality, the span name SHOULD be `{action}`.

The `{action}` SHOULD be one of the following values: `BUILD`, `RUN`, `SYNC`. If none of these values applies a custom value MAY be used.
```

Reply:
I made it similar to the http semconv where such placeholders are used.

Since we refer to several times (`if ... then ...`) it's more readable to use this placeholder instead of writing out the full attribute name.

Reply:
I improved the note like this in a8d7dea51d776dab86d45b042cbc82c5bb693dc3.